chunky/src/java/se/llbit/chunky/ui/render/tabs/CameraTab.java

```diff
@@ -265,6 +268,7 @@ private void updateShift() {
     fov.setName("Field of view (zoom)");
     fov.setRange(0.1, 180);
     fov.clampMin();
+    fov.setTooltip("Vertical field of view of the camera.");
```

Comment:
tbh. i don't see the benefit of the tooltip here. it doesn't provide any more information

Reply:
Yes, that control is self-explanatory, so I shall remove it.